src/QMCWaveFunctions/RotatedSPOs.cpp

```diff
@@ -35,8 +35,15 @@ RotatedSPOs::~RotatedSPOs() {}
 
 void RotatedSPOs::setRotationParameters(const std::vector<RealType>& param_list)
```

Comment:
In this case, the only part of the code that calls setRoatationParameters is from SPOSetBuilder, and that reads the optimization parameters into a vector of reals. rather than add additional functionality to the builder to handle real and complex types, this function should just take a list of reals as an argument. 

However, we had discussed (and subsequently forgot to change) params to use a better name like kappas or something, since those are the kappa parameters defined in the original orb opt papers. Then there wouldn't be any confusion about "params" and their types

Reply:
I think it is the first time we need to input complex number from xml. I feel the  caller side/builder need to handle the conversion. Not critical we can leave the code as it is now.